### PR TITLE
feat: add feedback on copy

### PR DESCRIPTION
**Pull request by mbhrznr**
fixes https://github.com/jsr-io/jsr/issues/730.

there hasn't been much of any feedback for the user that copying the code examples was successful.
this pr adds some immediate feedback in the form of a `check` icon, as seen in jsr's `CopyButton.tsx`.

~additionally `write_button` created and appended an empty `code` block, which is now removed.~

### CODE REVIEW

File: src/html/comrak_adapters.rs [hunk 1]
```diff
@@ -85,11 +85,11 @@ impl HighlightAdapter {
...
       output,
-      r#"<button class="context_button" data-copy="{}">{}</button>"#,
+      r#"<button class="context_button" data-copy="{}">{}{}</button>"#,
       html_escape::encode_double_quoted_attribute(source),
-      include_str!("./templates/icons/copy.svg")
-    )?;
-    write!(output, "<code>")
+      include_str!("./templates/icons/copy.svg"),
+      include_str!("./templates/icons/check.svg")
+    )
```
Comment (crowlKats) [inline src/html/comrak_adapters.rs:92]: missing opening `code` tag as there was before

Comment (mbhrznr) [reply src/html/comrak_adapters.rs:92]: the removal was intentional, as it created an empty `code` block.
should i reintroduce it?

Comment (mbhrznr) [reply src/html/comrak_adapters.rs]: fair enough! reverted that change, thanks for the hint!

**Review by crowlKats — APPROVED** [review]
LGTM, thanks!

**Pull request merged**